buildSrc/src/main/kotlin/com/datadog/gradle/Dependencies.kt

```diff
@@ -37,6 +41,8 @@ object Dependencies {
         const val DependencyVersion = "0.27.0"
         const val Dokka = "0.10.0"
         const val Bintray = "1.8.4"
+        const val Unmock = "0.7.5"
+        const val Robolectric = "4.4_r1-robolectric-r2" // Use lowest API
```

Comment:
Why do we need Robolectric ? I guess your intention was to use this to test your new strategy but then you found the "unmock" solution.

Reply:
We don't use the Robolectric runner, just the jar that implements the Android framework. 
Unmock let's you load only the required class from the Robolectric jar, which is used to unit test (among other things, it's needed to test against the Bundle class)